flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/JsonFunctionsTest.scala

```diff
@@ -125,4 +127,53 @@ class JsonFunctionsTest extends ExpressionTestBase {
     }
   }
 
+  @Test
+  def testJsonExists(): Unit = {
```

Comment:
Could you add a more complex test? Checking existence on a nested fields? e.g. `json_exist(f0, 'lax $.info.address[0].state')`. You can take [Sql Server](https://docs.microsoft.com/en-us/sql/t-sql/functions/json-value-transact-sql?view=sql-server-ver15#lax-mode-and-strict-mode) as an example.